src/main/java/fi/metatavu/metamind/bot/slots/utils/FinnishStringToNumberUtils.java

```diff
@@ -0,0 +1,81 @@
+package fi.metatavu.metamind.bot.slots.utils;
+
+/**
+ * Utility class to convert finnish word strings to numbers
+ * 
+ * @author Heikki Kurhinen
+ *
+ */
+public class FinnishStringToNumberUtils {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1118)